src/taskgraph/util/taskcluster.py

```diff
@@ -193,6 +193,35 @@ def find_task_id(index_path, use_proxy=False):
     return response.json()["taskId"]
 
 
+def find_task_id_batched(index_paths, use_proxy=False):
```

Comment:
Would be nice to include the docstring like you did for `status_task_batched` :)